src/main/java/revai/Listener.java

```diff
@@ -0,0 +1,49 @@
+package revai;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+import okhttp3.Response;
+import okhttp3.WebSocket;
+import okhttp3.WebSocketListener;
+import revai.models.streaming.Hypothesis;
+
+public class Listener extends WebSocketListener {
```

Comment:
does it need to be a public class?

with the RevAiWebsocketListener interface, this is not intended to be used by  users of the library, i think, its only used internally in the client. so i would make it a private nested class inside StreamingClient